.github/workflows/sanity_check.yml

```diff
@@ -32,7 +32,7 @@ jobs:
     - name: Download clang-format-diff.py
       uses: wei/wget@v1
       with:
-        args: https://raw.githubusercontent.com/llvm/llvm-project/master/clang/tools/clang-format/clang-format-diff.py
+        args: https://raw.githubusercontent.com/llvm-mirror/clang/master/tools/clang-format/clang-format-diff.py
```

Comment:
Thanks @riversand963 ! Was about to put out a PR when I noticed yours :) I believe we could stick with the official repo (i.e. no need to move to the unofficial mirror) as the script is still there under `https://raw.githubusercontent.com/llvm/llvm-project/main/clang/tools/clang-format/clang-format-diff.py`

Reply:
The only thing that's different is the branch name.

Reply:
Sure, let me update.